`npm run lint` failed

I'm submitting a PR, and found `lint` error. 

```bash
 npm run lint                                                                                                                             [094f795]

> tldr@1.7.1 lint /Users/taowang/lab/node.js/tldr-node-client
> eslint lib test bin/tldr


/Users/taowang/lab/node.js/tldr-node-client/lib/config.js
  66:21  error    Expected indentation of 4 spaces but found 20  indent
  67:21  error    Expected indentation of 4 spaces but found 20  indent
  68:21  error    Expected indentation of 4 spaces but found 20  indent
  69:21  error    Expected indentation of 4 spaces but found 20  indent
  70:21  error    Expected indentation of 4 spaces but found 20  indent
  71:21  error    Expected indentation of 4 spaces but found 20  indent
  72:21  error    Expected indentation of 4 spaces but found 20  indent
  73:21  error    Expected indentation of 4 spaces but found 20  indent
  74:21  error    Expected indentation of 4 spaces but found 20  indent
  75:21  error    Expected indentation of 4 spaces but found 20  indent
  76:21  error    Expected indentation of 4 spaces but found 20  indent
  77:21  error    Expected indentation of 4 spaces but found 20  indent
  78:21  error    Expected indentation of 4 spaces but found 20  indent
  79:21  error    Expected indentation of 4 spaces but found 20  indent
  80:21  error    Expected indentation of 4 spaces but found 20  indent
  81:21  error    Expected indentation of 4 spaces but found 20  indent
  82:21  error    Expected indentation of 4 spaces but found 20  indent
  83:21  error    Expected indentation of 4 spaces but found 20  indent
  84:21  error    Expected indentation of 4 spaces but found 20  indent
  85:21  error    Expected indentation of 4 spaces but found 20  indent
  86:21  error    Expected indentation of 4 spaces but found 20  indent
  87:21  error    Expected indentation of 4 spaces but found 20  indent
  88:21  error    Expected indentation of 4 spaces but found 20  indent
  89:21  error    Expected indentation of 4 spaces but found 20  indent
  90:21  error    Expected indentation of 2 spaces but found 20  indent
  91:3   warning  Unexpected 'todo' comment                      no-warning-comments

/Users/taowang/lab/node.js/tldr-node-client/lib/utils.js
  17:3  warning  Unexpected 'todo' comment  no-warning-comments

/Users/taowang/lab/node.js/tldr-node-client/bin/tldr
  77:3  warning  Unexpected 'todo' comment  no-warning-comments

✖ 28 problems (25 errors, 3 warnings)
```

I didn't touch those files or lines, I think it's might because the `lint` check had been turned off for some commits.

I didn't touch those files. To make sure it's clean, I cloned a new repo, and run `npm run lint` again, same errors:

```bash
$ git clone https://github.com/tldr-pages/tldr-node-client.git tldr-node-client2

Cloning into 'tldr-node-client2'...
remote: Counting objects: 1351, done.
remote: Compressing objects: 100% (48/48), done.
remote: Total 1351 (delta 14), reused 0 (delta 0), pack-reused 1303
Receiving objects: 100% (1351/1351), 442.40 KiB | 172.00 KiB/s, done.
Resolving deltas: 100% (830/830), done.
$ cd tldr-node-client2
$ npm run lint

> tldr@1.7.1 lint /Users/taowang/lab/node.js/tldr-node-client2
> eslint lib test bin/tldr


/Users/taowang/lab/node.js/tldr-node-client2/lib/config.js
  66:21  error    Expected indentation of 4 spaces but found 20  indent
  67:21  error    Expected indentation of 4 spaces but found 20  indent
  68:21  error    Expected indentation of 4 spaces but found 20  indent
  69:21  error    Expected indentation of 4 spaces but found 20  indent
  70:21  error    Expected indentation of 4 spaces but found 20  indent
  71:21  error    Expected indentation of 4 spaces but found 20  indent
  72:21  error    Expected indentation of 4 spaces but found 20  indent
  73:21  error    Expected indentation of 4 spaces but found 20  indent
  74:21  error    Expected indentation of 4 spaces but found 20  indent
  75:21  error    Expected indentation of 4 spaces but found 20  indent
  76:21  error    Expected indentation of 4 spaces but found 20  indent
  77:21  error    Expected indentation of 4 spaces but found 20  indent
  78:21  error    Expected indentation of 4 spaces but found 20  indent
  79:21  error    Expected indentation of 4 spaces but found 20  indent
  80:21  error    Expected indentation of 4 spaces but found 20  indent
  81:21  error    Expected indentation of 4 spaces but found 20  indent
  82:21  error    Expected indentation of 4 spaces but found 20  indent
  83:21  error    Expected indentation of 4 spaces but found 20  indent
  84:21  error    Expected indentation of 4 spaces but found 20  indent
  85:21  error    Expected indentation of 4 spaces but found 20  indent
  86:21  error    Expected indentation of 4 spaces but found 20  indent
  87:21  error    Expected indentation of 4 spaces but found 20  indent
  88:21  error    Expected indentation of 4 spaces but found 20  indent
  89:21  error    Expected indentation of 4 spaces but found 20  indent
  90:21  error    Expected indentation of 2 spaces but found 20  indent
  91:3   warning  Unexpected 'todo' comment                      no-warning-comments

/Users/taowang/lab/node.js/tldr-node-client2/lib/utils.js
  17:3  warning  Unexpected 'todo' comment  no-warning-comments

/Users/taowang/lab/node.js/tldr-node-client2/bin/tldr
  76:3  warning  Unexpected 'todo' comment  no-warning-comments

✖ 28 problems (25 errors, 3 warnings)


npm ERR! Darwin 16.3.0
npm ERR! argv "/usr/local/Cellar/node/7.4.0/bin/node" "/usr/local/bin/npm" "run" "lint"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! tldr@1.7.1 lint: `eslint lib test bin/tldr`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tldr@1.7.1 lint script 'eslint lib test bin/tldr'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the tldr package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint lib test bin/tldr
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs tldr
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls tldr
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/taowang/lab/node.js/tldr-node-client2/npm-debug.log
```

To avoid the system dependent issue, I did it inside a new docker container of image `buildpack-deps:xenial` and just installed `nvm` and `node`, the same error. 

```bash
$ docker run -it --rm buildpack-deps:xenial
root@01bbcd036758:/# curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 11329  100 11329    0     0  17865      0 --:--:-- --:--:-- --:--:-- 17869
=> Downloading nvm from git to '/root/.nvm'
=> Cloning into '/root/.nvm'...
remote: Counting objects: 6072, done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 6072 (delta 0), reused 0 (delta 0), pack-reused 6066
Receiving objects: 100% (6072/6072), 1.72 MiB | 347.00 KiB/s, done.
Resolving deltas: 100% (3774/3774), done.
Checking connectivity... done.
* (HEAD detached at v0.33.0)
  master
=> Compressing and cleaning up git repository
Counting objects: 6072, done.
Delta compression using up to 2 threads.
Compressing objects: 100% (6034/6034), done.
Writing objects: 100% (6072/6072), done.
Total 6072 (delta 4009), reused 1852 (delta 0)

=> Appending nvm source string to /root/.bashrc
=> bash_completion source string already in /root/.bashrc
=> Close and reopen your terminal to start using nvm or run the following to use it now:

export NVM_DIR="/root/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
root@01bbcd036758:/# source ~/.bashrc
root@01bbcd036758:/# nvm --version
0.33.0
root@01bbcd036758:/# cd
root@01bbcd036758:~# nvm install v7.4.0
Downloading and installing node v7.4.0...
Downloading https://nodejs.org/dist/v7.4.0/node-v7.4.0-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v7.4.0 (npm v4.0.5)
Creating default alias: default -> v7.4.0
root@01bbcd036758:~# git clone https://github.com/tldr-pages/tldr-node-client.git
Cloning into 'tldr-node-client'...
remote: Counting objects: 1351, done.
remote: Compressing objects: 100% (48/48), done.
remote: Total 1351 (delta 14), reused 0 (delta 0), pack-reused 1303
Receiving objects: 100% (1351/1351), 442.40 KiB | 176.00 KiB/s, done.
Resolving deltas: 100% (830/830), done.
Checking connectivity... done.
root@01bbcd036758:~# cd tldr-node-client/
root@01bbcd036758:~/tldr-node-client# npm i
npm WARN deprecated node-uuid@1.4.7: use uuid module instead

> husky@0.12.0 install /root/tldr-node-client/node_modules/husky
> node ./bin/install.js

husky
setting up hooks
done

tldr@1.7.1 /root/tldr-node-client
+-- chalk@1.1.3
| +-- ansi-styles@2.2.1
| +-- escape-string-regexp@1.0.5
| +-- has-ansi@2.0.0
| | `-- ansi-regex@2.1.1
| +-- strip-ansi@3.0.1
| `-- supports-color@2.0.0
+-- commander@2.9.0
| `-- graceful-readlink@1.0.1
+-- env-test@1.0.0
+-- eslint@3.13.1
| +-- babel-code-frame@6.20.0
| | `-- js-tokens@2.0.0
| +-- concat-stream@1.6.0
| | +-- inherits@2.0.3
| | +-- readable-stream@2.2.2
| | | +-- buffer-shims@1.0.0
| | | +-- process-nextick-args@1.0.7
| | | `-- util-deprecate@1.0.2
| | `-- typedarray@0.0.6
| +-- debug@2.6.0
| +-- doctrine@1.5.0
| | `-- isarray@1.0.0
| +-- escope@3.6.0
| | +-- es6-map@0.1.4
| | | +-- d@0.1.1
| | | +-- es5-ext@0.10.12
| | | +-- es6-iterator@2.0.0
| | | +-- es6-set@0.1.4
| | | +-- es6-symbol@3.1.0
| | | `-- event-emitter@0.3.4
| | +-- es6-weak-map@2.0.1
| | `-- esrecurse@4.1.0
| |   `-- estraverse@4.1.1
| +-- espree@3.3.2
| | +-- acorn@4.0.4
| | `-- acorn-jsx@3.0.1
| |   `-- acorn@3.3.0
| +-- estraverse@4.2.0
| +-- esutils@2.0.2
| +-- file-entry-cache@2.0.0
| | +-- flat-cache@1.2.2
| | | +-- circular-json@0.3.1
| | | +-- del@2.2.2
| | | | +-- globby@5.0.0
| | | | | +-- array-union@1.0.2
| | | | | | `-- array-uniq@1.0.3
| | | | | `-- arrify@1.0.1
| | | | +-- is-path-cwd@1.0.0
| | | | +-- is-path-in-cwd@1.0.0
| | | | | `-- is-path-inside@1.0.0
| | | | `-- pify@2.3.0
| | | `-- write@0.2.1
| | `-- object-assign@4.1.1
| +-- glob@7.1.1
| | +-- fs.realpath@1.0.0
| | +-- inflight@1.0.6
| | | `-- wrappy@1.0.2
| | +-- minimatch@3.0.3
| | | `-- brace-expansion@1.1.6
| | |   +-- balanced-match@0.4.2
| | |   `-- concat-map@0.0.1
| | `-- once@1.4.0
| +-- globals@9.14.0
| +-- ignore@3.2.0
| +-- imurmurhash@0.1.4
| +-- inquirer@0.12.0
| | +-- ansi-escapes@1.4.0
| | +-- cli-cursor@1.0.2
| | | `-- restore-cursor@1.0.1
| | |   +-- exit-hook@1.1.1
| | |   `-- onetime@1.1.0
| | +-- cli-width@2.1.0
| | +-- figures@1.7.0
| | +-- readline2@1.0.1
| | | +-- code-point-at@1.1.0
| | | +-- is-fullwidth-code-point@1.0.0
| | | | `-- number-is-nan@1.0.1
| | | `-- mute-stream@0.0.5
| | +-- run-async@0.1.0
| | +-- rx-lite@3.1.2
| | +-- string-width@1.0.2
| | `-- through@2.3.8
| +-- is-my-json-valid@2.15.0
| | +-- generate-function@2.0.0
| | +-- generate-object-property@1.2.0
| | | `-- is-property@1.0.2
| | +-- jsonpointer@4.0.1
| | `-- xtend@4.0.1
| +-- is-resolvable@1.0.0
| | `-- tryit@1.0.3
| +-- js-yaml@3.7.0
| | +-- argparse@1.0.9
| | | `-- sprintf-js@1.0.3
| | `-- esprima@2.7.3
| +-- json-stable-stringify@1.0.1
| | `-- jsonify@0.0.0
| +-- levn@0.3.0
| | +-- prelude-ls@1.1.2
| | `-- type-check@0.3.2
| +-- lodash@4.17.4
| +-- mkdirp@0.5.1
| | `-- minimist@0.0.8
| +-- natural-compare@1.4.0
| +-- optionator@0.8.2
| | +-- deep-is@0.1.3
| | +-- fast-levenshtein@2.0.6
| | `-- wordwrap@1.0.0
| +-- path-is-inside@1.0.2
| +-- pluralize@1.2.1
| +-- progress@1.1.8
| +-- require-uncached@1.0.3
| | +-- caller-path@0.1.0
| | | `-- callsites@0.2.0
| | `-- resolve-from@1.0.1
| +-- shelljs@0.7.6
| | +-- interpret@1.0.1
| | `-- rechoir@0.6.2
| |   `-- resolve@1.2.0
| +-- strip-bom@3.0.0
| +-- strip-json-comments@2.0.1
| +-- table@3.8.3
| | +-- ajv@4.10.4
| | | `-- co@4.6.0
| | +-- ajv-keywords@1.5.0
| | +-- slice-ansi@0.0.4
| | `-- string-width@2.0.0
| |   `-- is-fullwidth-code-point@2.0.0
| +-- text-table@0.2.0
| `-- user-home@2.0.0
+-- fs-extra@0.30.0
| +-- graceful-fs@4.1.11
| +-- jsonfile@2.4.0
| +-- klaw@1.3.1
| +-- path-is-absolute@1.0.1
| `-- rimraf@2.5.4
+-- husky@0.12.0
| +-- find-parent-dir@0.3.0
| +-- is-ci@1.0.10
| | `-- ci-info@1.0.0
| `-- normalize-path@1.0.0
+-- lodash.defaults@4.2.0
+-- lodash.get@4.4.2
+-- lodash.identity@3.0.0
+-- lodash.isempty@4.4.0
+-- lodash.sample@4.2.1
+-- lodash.unescape@4.0.1
+-- marked@0.3.6
+-- mocha@3.2.0
| +-- browser-stdout@1.3.0
| +-- debug@2.2.0
| | `-- ms@0.7.1
| +-- diff@1.4.0
| +-- glob@7.0.5
| +-- growl@1.9.2
| +-- json3@3.3.2
| +-- lodash.create@3.1.1
| | +-- lodash._baseassign@3.2.0
| | | +-- lodash._basecopy@3.0.1
| | | `-- lodash.keys@3.1.2
| | |   +-- lodash._getnative@3.9.1
| | |   +-- lodash.isarguments@3.1.0
| | |   `-- lodash.isarray@3.0.4
| | +-- lodash._basecreate@3.0.3
| | `-- lodash._isiterateecall@3.0.9
| `-- supports-color@3.1.2
|   `-- has-flag@1.0.0
+-- ms@0.7.2
+-- os-homedir@1.0.2
+-- request@2.75.0
| +-- aws-sign2@0.6.0
| +-- aws4@1.5.0
| +-- bl@1.1.2
| | `-- readable-stream@2.0.6
| +-- caseless@0.11.0
| +-- combined-stream@1.0.5
| | `-- delayed-stream@1.0.0
| +-- extend@3.0.0
| +-- forever-agent@0.6.1
| +-- form-data@2.0.0
| | `-- asynckit@0.4.0
| +-- har-validator@2.0.6
| | `-- pinkie-promise@2.0.1
| |   `-- pinkie@2.0.4
| +-- hawk@3.1.3
| | +-- boom@2.10.1
| | +-- cryptiles@2.0.5
| | +-- hoek@2.16.3
| | `-- sntp@1.0.9
| +-- http-signature@1.1.1
| | +-- assert-plus@0.2.0
| | +-- jsprim@1.3.1
| | | +-- extsprintf@1.0.2
| | | +-- json-schema@0.2.3
| | | `-- verror@1.3.6
| | `-- sshpk@1.10.2
| |   +-- asn1@0.2.3
| |   +-- assert-plus@1.0.0
| |   +-- bcrypt-pbkdf@1.0.0
| |   +-- dashdash@1.14.1
| |   | `-- assert-plus@1.0.0
| |   +-- ecc-jsbn@0.1.1
| |   +-- getpass@0.1.6
| |   | `-- assert-plus@1.0.0
| |   +-- jodid25519@1.0.2
| |   +-- jsbn@0.1.0
| |   `-- tweetnacl@0.14.5
| +-- is-typedarray@1.0.0
| +-- isstream@0.1.2
| +-- json-stringify-safe@5.0.1
| +-- mime-types@2.1.14
| | `-- mime-db@1.26.0
| +-- node-uuid@1.4.7
| +-- oauth-sign@0.8.2
| +-- qs@6.2.1
| +-- stringstream@0.0.5
| +-- tough-cookie@2.3.2
| | `-- punycode@1.4.1
| `-- tunnel-agent@0.4.3
+-- should@11.1.2
| +-- should-equal@1.0.1
| +-- should-format@3.0.2
| +-- should-type@1.4.0
| +-- should-type-adaptors@1.0.1
| `-- should-util@1.0.0
+-- sinon@1.17.7
| +-- formatio@1.1.1
| +-- lolex@1.3.2
| +-- samsam@1.1.2
| `-- util@0.10.3
|   `-- inherits@2.0.1
`-- unzip2@0.2.5
  +-- binary@0.3.0
  | +-- buffers@0.1.1
  | `-- chainsaw@0.1.0
  |   `-- traverse@0.3.9
  +-- fstream@0.1.31
  | `-- graceful-fs@3.0.11
  |   `-- natives@1.1.0
  +-- match-stream@0.0.2
  | `-- readable-stream@1.0.34
  |   `-- isarray@0.0.1
  +-- pullstream@0.4.1
  | +-- over@0.0.5
  | +-- readable-stream@1.0.34
  | | `-- isarray@0.0.1
  | `-- slice-stream@1.0.0
  |   `-- readable-stream@1.0.34
  |     `-- isarray@0.0.1
  +-- readable-stream@1.0.34
  | +-- core-util-is@1.0.2
  | +-- isarray@0.0.1
  | `-- string_decoder@0.10.31
  `-- setimmediate@1.0.5

root@01bbcd036758:~/tldr-node-client# npm run lint

> tldr@1.7.1 lint /root/tldr-node-client
> eslint lib test bin/tldr


/root/tldr-node-client/lib/config.js
  66:21  error    Expected indentation of 4 spaces but found 20  indent
  67:21  error    Expected indentation of 4 spaces but found 20  indent
  68:21  error    Expected indentation of 4 spaces but found 20  indent
  69:21  error    Expected indentation of 4 spaces but found 20  indent
  70:21  error    Expected indentation of 4 spaces but found 20  indent
  71:21  error    Expected indentation of 4 spaces but found 20  indent
  72:21  error    Expected indentation of 4 spaces but found 20  indent
  73:21  error    Expected indentation of 4 spaces but found 20  indent
  74:21  error    Expected indentation of 4 spaces but found 20  indent
  75:21  error    Expected indentation of 4 spaces but found 20  indent
  76:21  error    Expected indentation of 4 spaces but found 20  indent
  77:21  error    Expected indentation of 4 spaces but found 20  indent
  78:21  error    Expected indentation of 4 spaces but found 20  indent
  79:21  error    Expected indentation of 4 spaces but found 20  indent
  80:21  error    Expected indentation of 4 spaces but found 20  indent
  81:21  error    Expected indentation of 4 spaces but found 20  indent
  82:21  error    Expected indentation of 4 spaces but found 20  indent
  83:21  error    Expected indentation of 4 spaces but found 20  indent
  84:21  error    Expected indentation of 4 spaces but found 20  indent
  85:21  error    Expected indentation of 4 spaces but found 20  indent
  86:21  error    Expected indentation of 4 spaces but found 20  indent
  87:21  error    Expected indentation of 4 spaces but found 20  indent
  88:21  error    Expected indentation of 4 spaces but found 20  indent
  89:21  error    Expected indentation of 4 spaces but found 20  indent
  90:21  error    Expected indentation of 2 spaces but found 20  indent
  91:3   warning  Unexpected 'todo' comment                      no-warning-comments

/root/tldr-node-client/lib/utils.js
  17:3  warning  Unexpected 'todo' comment  no-warning-comments

/root/tldr-node-client/bin/tldr
  76:3  warning  Unexpected 'todo' comment  no-warning-comments

✖ 28 problems (25 errors, 3 warnings)


npm ERR! Linux 4.8.15-moby
npm ERR! argv "/root/.nvm/versions/node/v7.4.0/bin/node" "/root/.nvm/versions/node/v7.4.0/bin/npm" "run" "lint"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! tldr@1.7.1 lint: `eslint lib test bin/tldr`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tldr@1.7.1 lint script 'eslint lib test bin/tldr'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the tldr package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint lib test bin/tldr
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs tldr
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls tldr
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /root/tldr-node-client/npm-debug.log
root@01bbcd036758:~/tldr-node-client#
```

--- a/lib/config.js
+++ b/lib/config.js
@@ -1,8 +1,8 @@
 'use strict';
 
 const defaults = require('lodash.defaults');
-const fs        = require('fs');
-const path      = require('path');
+const fs = require('fs');
+const path = require('path');
 const osHomedir = require('os-homedir');
 
 let config = null;
@@ -19,7 +19,7 @@ exports.get = () => {
 };
 
 function load() {
-  const DEFAULT  = path.join(__dirname, '..', 'config.json');
+  const DEFAULT = path.join(__dirname, '..', 'config.json');
   const CUSTOM = path.join(osHomedir(), '.tldrrc');
 
   let defaultConfig = JSON.parse(fs.readFileSync(DEFAULT));
@@ -39,13 +39,13 @@ function load() {
   let merged = defaults(customConfig, defaultConfig);
   // Validating the theme settings
   let errors = Object.keys(!merged.themes ? {} : merged.themes).map(
-      (key) => {
-        return validateThemeItem(merged.themes[key], key);
-      }
-    );
+    (key) => {
+      return validateThemeItem(merged.themes[key], key);
+    }
+  );
   errors.push(validatePlatform(merged.platform));
   // Filtering out all the null entries
-  errors = errors.filter((item) => {return item !== null;});
+  errors = errors.filter((item) => { return item !== null; });
 
   if (errors.length > 0) {
     throw new Error('Error in .tldrrc configuration:\n' + errors.join('\n'));
@@ -63,36 +63,36 @@ function validatePlatform(os) {
 
 function validateThemeItem(field, key) {
   let validValues = ['',
-                    'reset',
-                    'bold',
-                    'dim',
-                    'italic',
-                    'underline',
-                    'inverse',
-                    'hidden',
-                    'black',
-                    'red',
-                    'green',
-                    'yellow',
-                    'blue',
-                    'magenta',
-                    'cyan',
-                    'white',
-                    'gray',
-                    'bgBlack',
-                    'bgRed',
-                    'bgGreen',
-                    'bgYellow',
-                    'bgBlue',
-                    'bgMagenta',
-                    'bgCyan',
-                    'bgWhite'
-                    ];
+    'reset',
+    'bold',
+    'dim',
+    'italic',
+    'underline',
+    'inverse',
+    'hidden',
+    'black',
+    'red',
+    'green',
+    'yellow',
+    'blue',
+    'magenta',
+    'cyan',
+    'white',
+    'gray',
+    'bgBlack',
+    'bgRed',
+    'bgGreen',
+    'bgYellow',
+    'bgBlue',
+    'bgMagenta',
+    'bgCyan',
+    'bgWhite'
+  ];
   // TODO: change this to return all errors in a field
   for (let item in field) {
     if (field.hasOwnProperty(item)) {
-      let tokens = field[item].replace(/\s+/g,'').split(',');
-      for (let i=0; i < tokens.length; i++) {
+      let tokens = field[item].replace(/\s+/g, '').split(',');
+      for (let i = 0; i < tokens.length; i++) {
         if (validValues.indexOf(tokens[i]) < 0) {
           return 'Invalid theme value : ' + tokens[i] + ' in ' + key + ' theme';
         }

